### PR TITLE
V2/fix nav mobile

### PR DESCRIPTION
**Pull request by dospore**
Fix mobile nav
https://www.loom.com/share/8fab4e8a108c48ed90d58569e9fc001f

### CODE REVIEW

File: components/Nav/Navbar/NetworkDropdown.tsx [hunk 1]
```diff
@@ -55,6 +75,7 @@ const NetworkPreview: React.FC<{
     networkName: string;
     supported: boolean;
 }> = ({ networkID, networkName, supported }) => {
+    console.log('network', networkID, supported);
```
Comment (dweberdev) [inline components/Nav/Navbar/NetworkDropdown.tsx:78]: Should we keep this console.log?

**Review by dweberdev — DISMISSED** [review]
lgtm

File: components/Nav/Navbar/NetworkDropdown.tsx (hunk 1 above)
Comment (dospore) [reply components/Nav/Navbar/NetworkDropdown.tsx]: nah thankyou, removed

File: components/General/Container/index.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styled from 'styled-components';
+import { classNames } from '~/utils/helpers';
+
+export const Container = styled(({ className, children }) => (
```
[the remark this answers is not in the document]
Comment (dweberdev) [reply components/General/Container/index.tsx:5]: nvm I see you added

**Review by dweberdev — APPROVED** [review]
lgtm

**Pull request merged**